### DISCUSSION TITLE
API docs?

### DISCUSSION THREAD

**Discussion by joostfaassen** [ente-io/ente#2311] [Q&A]
Hey guys, Ente is awesome, and I'd like to explore the option of using it as a storage layer for other apps.
Does ente have any documentations about how a client would interact with the ente server? I know there's client code in the repo in go. are there any other sources of info / docs / openapi specs that we could use to implement a client in other programming languages?

**Comment by siepkes** [comment]
I was also looking for info on this. Specifically OpenAPI specs would be great!

**Comment by tafli** [comment]
Would be great having Ente a storage layers for other apps. I think especially about a layer/plugin for CMS like Wordpress and others. Currently, we use a CMS with a Flickr integrations to show albums/photos on our website. Would love to be able to use Ente for this.
